app/views/deed/_deeds.html.slim

```diff
@@ -12,8 +12,5 @@
 
 small.legend
   =link_to 'Show More',
-    { :controller => 'deed',
-      :action => 'list',
-      :collection_id => @collection ? @collection.id : nil,
-      :user_id => options[:user_id] },
-    class: 'button outline round'
\ No newline at end of file
+    deed_list_path(:user_id => options[:user_id]),
```

Comment:
This looks like it formerly worked both from the user context and from the collection context.  We should make sure we test (and understand how to test) the collection context.

Reply:
@saracarl I just restored the previous functionality where we pass `collection_id`.

I removed that because I was experiencing breaks during the upgrade, and removing `collection_id` worked as a workaround.

It would still be nice to know how to trigger the collection context.

Good find, thanks.